source/controlTypes.py

```diff
@@ -662,21 +664,46 @@ class OutputReason(Enum):
 REASON_CARET = OutputReason.CARET
 REASON_ONLYCACHE = OutputReason.ONLYCACHE
 
+
+class IS_CURRENT(Enum):
+	"""Values to use within NVDA to denote 'current' values.
+	These describe if an item is the current item within a particular kind of selection.
+	EG aria-current
+	"""
+	NO = "false"
+	Yes = "true"
+	PAGE = "page"
+	STEP = "step"
+	LOCATION = "location"
+	DATE = "date"
+	TIME = "time"
+
+	def getTranslationForUI(self):
+		try:
+			return isCurrentLabels[self]
+		except KeyError:
+			log.debugWarning(f"No translation mapping for: {self}")
+			# there is a value for 'current' but NVDA hasn't learned about it yet,
+			# at least describe in the general sense that this item is 'current'
+			return isCurrentLabels[IS_CURRENT.Yes]
+
+
 #: Text to use for 'current' values. These describe if an item is the current item 
-#: within a particular kind of selection.
-isCurrentLabels: Dict[Union[bool, str], str] = {
+#: within a particular kind of selection. EG aria-current
+isCurrentLabels: Dict[IS_CURRENT, str] = {

```

Comment:
Could this be moved onto the enum?, as it is now strictly private to the enum?

Reply:
Yes, I think so.

Reply:
This was a bit of a rabbit hole. Turns out moving `isCurrentLabels` into the class causes it to be considered an Enum Member. This ends up being a bit confusing. I think it is better outside of the class, but I have prefixed an underscore to mark it as module private.

Reply:
Ah, sounds ok to me.